android/build.gradle

```diff
@@ -1,8 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+apply plugin: 'com.android.library'
+apply from: 'gradle-maven-push.gradle'
+
 buildscript {
   repositories {
-    jcenter()
+      mavenLocal()
+      jcenter()
+      maven {
+          url "$projectDir/../../react-native/android"
```

Comment:
Actually no, my change is correct. In this case, `$projectDir` refers to `<install location>/node_modules/react-native-maps/android`, so `$projectDir/../../react-native/android` resolves to `<install location>/node_modules/react-native/android`

Putting `url "$projectDir/../node_modules/react-native/android"` resolves to `<install location>/node_modules/react-native-maps/node_modules/react-native/android`, a path which may not, and probably won't exist on end-user installs of this plugin. This is one of the things which causes Gradle to fetch the package from Maven, which isn't optimal.